Miniforge/construct.yaml

```diff
@@ -1,10 +1,9 @@
-name: Miniforge3
```

Comment:
The directory is still Miniforge3

Reply:
It's Miniforge now :-)

Reply:
Ah ok. renames are still the same "file"